apps/beeai-server/src/beeai_server/configuration.py

```diff
@@ -26,7 +28,9 @@ class Configuration(BaseSettings):
     model_config = SettingsConfigDict(env_file=".env", env_file_encoding="utf-8", env_nested_delimiter="__")
     logging: LoggingConfiguration = LoggingConfiguration()
     provider_config_path: Path = Path.home() / ".beeai" / "providers.yaml"
+    cache_dir: Path = Path.home() / ".beeai" / "cache"
```

Comment:
Perhaps we could be nice and switch to XDG standard dirs for config/state/cache (https://specifications.freedesktop.org/basedir-spec/latest/)? Looking at my home dir, it looks like many tools use them even on Mac (i.e. `pnpm`) -- and many don't and just go into `~/.<name>` (i.e. `npm`).

Reply:
I thought about it, but overall I think it's better for the users to just have everything in one place so that you can do `rm -rf ~./beeai` if things go wrong and you have to reset. Also makes for a nicer troubleshooting guide :D  

I'd keep it as is now but consider moving both eventually if we want to be compliant - config to:
`~/.config/beeai` and cache to `~/.cache/beeai`